package.json

```diff
@@ -5,12 +5,13 @@
   "scripts": {
     "start": "node server/index.js",
     "server": "nodemon server/index.js",
-    "client": "cd client && npm start",
+    "client": " cd client  && npm run build",
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "eslint": "eslint ./server",
     "eslint:fix": "eslint --fix ./server",
     "test": "jest ./tests",
-    "inst": "npm i && cd client && npm i"
+    "inst": "npm i && cd client && npm i",
+    "heroku-postbuild": "cd client && npm i && npm run build"
```

Comment:
what about this delete or stay it?
because you said if I'd like to add a client build script for heroku, you think that maybe it should be done in a separate script:
"client:build" : "cd client && npm run build"

Reply:
hmm, maybe i am not understanding correctly. why did you change the client script to run the build script. were you using it somewhere? @shaima96

Reply:
From my understanding the client script should be our development script and the build script should only be run for deployment:

https://facebook.github.io/create-react-app/docs/production-build

I could be misunderstanding so feel free to correct me if I am wrong!

Reply:
In previous project we put a script as :
    "build": "cd client && npm run build",
    "client": "cd client && npm run start",
    "heroku-postbuild": "cd client && npm install && npm run build",

Reply:
@shaima96 yes this looks great! if you could change it to that, that would be perfect!
```
"build": "cd client && npm run build",
"client": "cd client && npm run start",
"heroku-postbuild": "cd client && npm install && npm run build", 
```

what i was confused by in the first instance was the change of our client script from
```"client": "cd client && npm run start",```
to
```"client": "cd client && npm run build",```
but you've changed that back now!

sorry for all the confusion.

Reply:
Don't worry 
plz, check the `PR` after changed in `package.json`.